tests/callbacks/test_progress_bar.py

```diff
@@ -558,3 +559,27 @@ def _test_progress_bar_max_val_check_interval(
     total_val_batches = total_val_batches * val_checks_per_epoch
     if trainer.is_global_zero:
         assert trainer.progress_bar_callback.main_progress_bar.total == total_train_batches + total_val_batches
+
+
+def test_get_progress_bar_metrics(tmpdir: str):
+    class TestProgressBar(ProgressBar):
+        def get_metrics(self, trainer: Trainer, model: LightningModule):
+            items = super().get_metrics(trainer, model)
+            items.pop("v_num", None)
+            return items
+
+    progress_bar = TestProgressBar()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        callbacks=[progress_bar],
+        progress_bar_refresh_rate=None,
+        max_epochs=1,
+        overfit_batches=5,
```

Comment:
let's train with as few steps as possible. 

```suggestion
        fast_dev_run=True,
```